jsapp/js/components/mapSettings.es6

```diff
@@ -339,10 +362,22 @@ class MapSettings extends React.Component {
                 </div>
               </bem.FormModal__item>
             )}
+            {activeTab === 'querylimit' && (
+              <bem.FormModal__item>
+                <div className="map-settings__querylimit">
+                  {t('By default the map is limited to the 5000 most recent submissions. You can temporarily increase this limit to a different value. Note that this is reset whenever you reopen the map.')}
+                  <p className="change-limit-warning">Warning: Displaying a large number of points requires a lot of memory.</p>
+                  <form onInput={this.updateSliderValue}>
+                    <input id="limit-slider" className="change-limit-slider" type="range" step="1000" min="1000" max="30000" value={queryLimit} onChange={this.queryLimitChange}/>  
+                    <output id="limit-slider-value" className="change-limit-slider-value" htmlFor="limit-slider">{queryLimit}</output>
```

Comment:
This comment is more nitpicky than a change request, but instead of this read-only element to display queryLimit, you could have a `<input type="number"…` (with `min`, `max`, `step` and some `onChange` callback value validation) for a better UX. Slider is nice, but some users will want to have exact value there, and it is easier to type or paste it than handle the slider :)

Reply:
And if you use `input`, don't forget to remove `.change-limit-slider-value` styles.

Reply:
should i put both in for user testing purposes? or should i make a new branch for either or should i just commit to one ;)